BROWSER_SUPPORT.md

```diff
@@ -13,23 +13,20 @@ combinations are supported.
 - Windows 7
 - Windows 8.1
 - Windows 10
-- OSX Catalina (13) - \_NB: This doesn't like working on certain pages, there are several issues with `safaridriver`
-- OSX Mojave (12)
+- OSX Mojave (12) - We **don't** support Catalina (13), as it is too flaky (Big Sur will be supported).
```

Comment:
This is more about framing, but we could we frame this document so that it's clear that we are talking about _automated testing_ support? I don't want to accidentally imply that the components wouldn't work fine in these OS combinations. This is about what we test against in BrowserStack right?

Reply:
Actually looking at the full doc I can see you've kinda covered that in the preamble. So 👍